src/content/cloud/container-resources.mdx

```diff
@@ -0,0 +1,63 @@
+sidebar_label: Container Resources
+id: container-resources
+---
+
+import Image from '@theme/Image';
+
+Okteto leverages the concept of Kubernetes resources to decide on which node to place each container.
+When you specify a resource `limit` for a container, Okteto enforces those limits so that the running container is not allowed to use more of that resource than the limit you set.
+Okteto also reserves at least the `request` amount of that system resources specifically for that container to use.
+Follow [this link](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) to know more about how Kubernetes resources work.
+
+## Best Practices
+
+Okteto monitors the CPU and Memory usage of your development environments to balance containers across all the nodes in the cluster.
+However, it's not capable of evicting containers when a node becomes overloaded.
+To avoid this situation, it's strongly recommended that you configure your CPU and Memory `requests` to align with your container's actual resource utilization.
+By doing so, you ensure that your containers always have sufficient resources available for optimal development speed.
+
+## Resource Consumption Warnings
+
+The Okteto UI shows warnings to assist you in setting the right resources for your development environments.
+If a development environment is utilizing over 50% of its CPU or Memory requests, you'll receive a yellow warning:
+
+<p align="center"><Image src="/docs/cloud/images/container-resources-yellow.png" alt="container resources yellow warning" width="650" /></p>
+
+If it's using more than 100% of its CPU and Memory requests, you'll receive a red warning:
+
+<p align="center"><Image src="/docs/cloud/images/container-resources-red.png" alt="container resources red warning" width="650" /></p>
+
+
+## Access your Metrics with kubectl
+
+If you require more detailed data on your container resource consumption, you can use the `kubectl top pods` command.
+
+In order to use `kubectl top pods`, follow these steps:
+
+- [Install](getting-started/#installing-okteto-cli) the Okteto CLI
+- Configure your Okteto context with `okteto ctx`
+- Download your kubectl credentials with `okteto kubeconfig`
+
+And run `kubectl top pods` to access your container's metrics:
+
+```console
+$ kubectl top pod
+```
+
+```
+NAME                        CPU(cores)   MEMORY(bytes)
+api-7cc77bcdcb-pm72j        89m          66Mi
+frontend-78b8b698cf-5wb5p   1m           2Mi
+mongodb-0                   5m           160Mi
+```
+
+## Configure your Development Environment Resources
+
+There are three differenct ways to configure the container resources for your development environments:
+
+- [Kubernetes manifest](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+- [Docker Compose](/reference/compose/#resources-object-optional)
+- [Okteto Manifest](/reference/manifest/#resources-object-optional)
```

Comment:
can we add small samples on this doc? (at least for the okteto way?). 

Also, giving someone three ways of doing something without clarifying which is the recommended way makes it confusing (in my opinion). Could we add a single recommendation (the manifest) and then maybe, as a subsection, say something like "if this is not ideal, then you can also do it at X or Y"?

Reply:
The recommendation is to define requests in both, the okteto manifest, and your deployment manifests (either kubernetes or docker compose). I can add samples, but I would mention the three places